compiler/p/codegen/OMRTreeEvaluator.cpp

```diff
@@ -2576,22 +2576,28 @@ TR::Register *OMR::Power::TreeEvaluator::vloadEvaluator(TR::Node *node, TR::Code
    switch(node->getDataType())
      {
      case TR::VectorInt8:
+         cg->comp()->target().cpu.isAtLeast(OMR_PROCESSOR_PPC_P9) ? opcode = TR::InstOpCode::lxvb16x : opcode = TR::InstOpCode::lxvw4x;
```

Comment:
Please don't put assignments as sub-expressions of a ternary. Would be much better to use:

```suggestion
         opcode = cg->comp()->target().cpu.isAtLeast(OMR_PROCESSOR_PPC_P9) ? TR::InstOpCode::lxvb16x : TR::InstOpCode::lxvw4x;
```

Similar problems appear in a few other places.